src/constants/api.ts

```diff
@@ -43,12 +43,19 @@ export const techDep = [9, 10, 12]
 export const totalDays = 10
 
 // date format for Queries
-let sprintNumStart = 32
-let sprintNumEnd = 33
+let sprintNumStart = dayjs().week()
+let sprintNumEnd = sprintNumStart +1
+
+if( sprintNumStart %2 !==0){
```

Comment:
```suggestion
// Si on est sur une semaine impaire, le sprint qui nous intéresse est le prochain (démarrant la semaine suivante)
if( sprintNumStart %2 !==0){
```
Juste histoire de ne pas oublier pourquoi on a ce if

Reply:
Exact! D'ailleurs je pense, même étant sur une semaine paire, ne vaut il pas mieux aller sur une +2 ?
ça serait à voir peut être avec les pm ?

Reply:
Oui tu as raison 👍